<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QwMainWindow</class>
 <widget class="QMainWindow" name="QwMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QwMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionembedded_widget_action"/>
   <addaction name="actionindependent_widget_action"/>
   <addaction name="separator"/>
   <addaction name="actionembedded_mainwindow_action"/>
   <addaction name="actionindependent_mainwindow_action"/>
   <addaction name="separator"/>
   <addaction name="quit_action"/>
  </widget>
  <action name="actionembedded_widget_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons/preview1.bmp</normaloff>:/img/icons/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>embedded widget</string>
   </property>
  </action>
  <action name="actionindependent_widget_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons/print1.bmp</normaloff>:/img/icons/print1.bmp</iconset>
   </property>
   <property name="text">
    <string>independent widget</string>
   </property>
  </action>
  <action name="actionembedded_mainwindow_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons/saveall1.bmp</normaloff>:/img/icons/saveall1.bmp</iconset>
   </property>
   <property name="text">
    <string>embedded mainwindow</string>
   </property>
  </action>
  <action name="actionindependent_mainwindow_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons/upfold1.bmp</normaloff>:/img/icons/upfold1.bmp</iconset>
   </property>
   <property name="text">
    <string>independent mainwindow</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons/undo.bmp</normaloff>:/img/icons/undo.bmp</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
